translate/src/context/TranslationContext.tsx

```diff
@@ -35,14 +29,11 @@ const LLMTranslationContext = createContext<LLMTranslationContextType>({
 
 export const LLMTranslationProvider: React.FC = ({ children }) => {
   const stateRef = useRef(new WeakMap<MachineryTranslation, SelState>());
-  const [version, setVersion] = useState(0); // Counter to trigger re-renders
+  const [_, setVersion] = useState(0); // Counter to trigger re-renders
```

Comment:
The unused variable can be elided completely.
```suggestion
  const [, setVersion] = useState(0); // Counter to trigger re-renders
```